src/style.css

```diff
@@ -1,3 +1,67 @@
+:root {
+  --blue: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  background-color: aqua;
+  color: var(--blue);
+}
+
+.header {
+  width: 100%;
+  min-width: 1024px;
+  display: flex;
+  background-color: white;
+}
```

Comment:
The 'header' class has a fixed width specified (min-width: 1024px). According to the checklist, fixing container sizes should be avoided unless specifically required. This could lead to overflow or accidental scroll bars.